_rules/explicit-SVG-image-has-name-7d6734.md

```diff
@@ -36,13 +36,13 @@ This rule assumes that the presence of one of the roles outlined in the applicab
 
 ## Accessibility Support
 
-The [HTML Accessibility API Mappings](https://www.w3.org/TR/html-aam-1.0/#html-element-role-mappings) specify that the `<svg>` element has an implicit role of `graphics-document`. However browser support for the `graphics-document` role and the [SVG Accessibility API Mappings][] is inconsistent.
+The [HTML Accessibility API Mappings](https://www.w3.org/TR/html-aam-1.0/#html-element-role-mappings) specify that the `svg` element has an implicit role of `graphics-document`. However browser support for the `graphics-document` role and the [SVG Accessibility API Mappings][] is inconsistent.
 
 This rule is limited to the explicit use of roles, as a clear indication that content should convey meaning, until the [SVG Accessibility API Mappings][] is more stable and browser support is more consistent.
 
-Browser and assistive technology support for SVG `<title>` and `<desc>` elements is currently inconsistent. Using WAI ARIA in combination with the `img` role for non-decorative `<svg>` elements significantly improves accessibility browser support.
+Browser and assistive technology support for SVG `title` and `<desc>` elements is currently inconsistent. Using WAI ARIA in combination with the `img` role for non-decorative `svg` elements significantly improves accessibility browser support.
```

Comment:
Somehow, `<desc>` has not been caught, which indicates maybe a problem with the test…

Reply:
I use the library `html-tags` -> https://www.npmjs.com/package/html-tags to figure out if a given piece of string is a html tag. `desc` seems to not be here - https://github.com/sindresorhus/html-tags/blob/master/html-tags.json